redis.c

```diff
@@ -551,6 +551,10 @@ static void add_class_constants(zend_class_entry *ce, int is_cluster TSRMLS_DC)
     zend_declare_class_constant_long(ce, ZEND_STRL("SERIALIZER_IGBINARY"), REDIS_SERIALIZER_IGBINARY TSRMLS_CC);
 #endif
 
+#ifdef HAVE_REDIS_IGBINARY
```

Comment:
Jeez.  In my haste I missed that; compiled with both igbinary and msgpack, so there were no issues.  Sorry about that.  Will amend the commit now.